docs/source/mnn/index.rst

```diff
@@ -0,0 +1,29 @@
+sherpa-mnn
```

Comment:
Please follow the README.md in
https://github.com/k2-fsa/sherpa/tree/master/docs
to check the documentation in your browser.